packages/stargate/src/stargateclient.ts

```diff
@@ -155,6 +195,28 @@ export class StargateClient {
     this.tmClient.disconnect();
   }
 
+  public async broadcastTx(tx: cosmos.tx.Tx): Promise<BroadcastTxResponse> {
+    const params = {
+      tx: cosmos.tx.Tx.encode(tx).finish(),
```

Comment:
I'd just take a serialized `Uint8Array` here and keep all the encoding at some encoding place.